### PR TITLE
[MB-4308] Move 101 page

### PR DESCRIPTION
**Pull request by suzubara**
## Description

This PR uses the new MilmoveCard component and updates the Moving 101 page with new content.

## Reviewer Notes

To test, you will need to log in as a service member and add your first shipment. The Moving 101 page should display ahead of selecting the shipment type. Alternatively, you can log in as a service member who already has profile/orders info and navigate to: `/moves/<move ID>/moving-info`

## Setup

To start the application locally:
```
make server_run
make client_run
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-4308) for this change

## Screenshots

image

image

### CODE REVIEW

**Review by amanda5iu — APPROVED** [review]
Looks great!

File: src/pages/MyMove/MovingInfo.jsx [hunk 1]
```diff
@@ -1,31 +1,97 @@
...
 import SectionWrapper from 'components/Customer/SectionWrapper';
+import MilmoveCard from 'components/Customer/MilmoveCard/MilmoveCard';
 
 export const MovingInfo = () => {
   return (
     <>
-      <h1 data-testid="shipmentsHeader">Figure out your shipments</h1>
-      <p>Handy tips as you decide how to move</p>
+      <h1 data-testid="shipmentsHeader">Tips for planning your shipments</h1>
+      <Alert type="info" heading="You can move 7,000 lbs for free" noIcon>
```
Comment (kimallen) [inline src/pages/MyMove/MovingInfo.jsx:15]: I know it's probably not in the scope of this story, but do we have a story to link the number of pounds to their actual weight allowance?

**Review by kimallen — COMMENTED** [review]
Everything looks good to me!  Can you double check if the weight in the alert should be dynamic for this story or whether there's another story for that?  Otherwise I can approve.

File: src/pages/MyMove/MovingInfo.jsx (hunk 1 above)
Comment (kimallen) [reply src/pages/MyMove/MovingInfo.jsx:15]: Update: PM is aware and will find or create a story.

**Review by kimallen — APPROVED** [review]

**Pull request merged**